frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1869,9 +1905,25 @@ function ReaderFooter:_updateFooterText(force_repaint, force_recompute)
     end
 end
 
+function ReaderFooter:onTocReset()
+    self:setTocMarkers(true)
+    self:onUpdateFooter()
```

Comment:
I'm not sure how soon after a toc reset the TOC was filled again before this PR (might be that it was delayed after a repaint, dunno). But with this, is will be immediately filled.
(I can't answer that question after a quick look at the code, so, just asking in case someone get an idea :)

Reply:
Note that `ReaderRolling:updatePos()` had before:
```
        self.ui:handleEvent(Event:new("UpdateToc"))
        self.view.footer:setTocMarkers(true)
        self.view.footer:onUpdateFooter()
```
so nothing should have changed in that particular chain of events.